src/main/java/modul1/Order.java

```diff
@@ -0,0 +1,54 @@
+package modul1;
+
+import java.util.List;
+
+public class Order {
+
+    private String id;
+    private List<Product> products;
+
+    public Order(String id, List<Product> products2) {
+    }
```

Comment:
Hier solltest du die Parameter den Attributen der Order Klasse zuweisen
```suggestion
    public Order(String id, List<Product> products) {
         this.id = id;
         this.products = products;
    }
```